test/OpenTelemetry.Tests/Logs/LoggerProviderTests.cs

```diff
@@ -0,0 +1,85 @@
+// <copyright file="LoggerProviderTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+#if !NET461
+using System;
+
+using OpenTelemetry.Resources;
+
+using Xunit;
+
+namespace OpenTelemetry.Logs.Tests
+{
+    public sealed class LoggerProviderTests
+    {
+        [Fact]
+        public void VerifyDefaultBehavior()
+        {
+            var options = new OpenTelemetryLoggerOptions();
```

Comment:
While this maybe okay, I'd suggest to add tests which would test what would be doing - (more like integration test) - Users won't new up a Provider and pass options to it, they only deal with loggerfactory/loggers.